env/vars.go

```diff
@@ -7,6 +7,21 @@ type Vars struct {
 	ignoreCase bool
 }
 
+func varsFromEnviron(environ []string, ignoreCase bool, removeKey func(string) bool) *Vars {
+	vars := NewVars(nil, ignoreCase)
+	for _, kv := range environ {
```

Comment:
Same comment about `environ`.
I think if we're trying to make it a short name, it's better to use `env`, but personally I'm also fine with `environment`

Reply:
I went with `environ` as a nod to [os.Environ](https://golang.org/pkg/os/#Environ) which returns the environment in the format expected here. I originally avoid `env` because it matches the package name, but on second thought there is no practical problem with that and the linter doesn't seem to mind so maybe it's kosher.

Reply:
ohhh...I wasn't aware of `os.Environ`.
I'm just more used to `env` but if the convention is `environ`, I'm totally ok with that.

Reply:
Just saw that you changed it to `env`. Thanks!